### PR TITLE
feat(RHINENG-13865): Create content for OS versions

### PR DESCRIPTION
**Pull request by johnsonm325**
To test:
- manually [set feature flag in code to true](https://github.com/RedHatInsights/registration-assistant/blob/master/src/Routes.js#L18)
- navigate to registration assistant
- go through steps and match each selected operating system in step 2 to the mock-ups

### CODE REVIEW

**Review by adonispuente — CHANGES_REQUESTED** [review]
Just some small nitpicks, main issue is the padding, otherwise everything else seems good.

In Step 1, 'Show selected activation key' is missing. Though I know this PR is for step 
2, just wanted to bring awareness.

Screenshot 2024-11-19 at 4 58 20 PM

Theres incorrent padding on the rendered content for all OS except Rhel 9 or Later
Screenshot 2024-11-19 at 5 00 14 PM

File: src/Components/RegAssistCodeBlock/RegAssistCodeBlock.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
   ClipboardCopyButton,
 } from '@patternfly/react-core';
 
-const RegAssistCodeBlock = ({ code = [] }) => {
+const RegAssistCodeBlock = ({ code = [], classname }) => {
```
Comment (adonispuente) [inline src/Components/RegAssistCodeBlock/RegAssistCodeBlock.js:10]: Why are we passing in this classname prop?

Comment (johnsonm325) [reply src/Components/RegAssistCodeBlock/RegAssistCodeBlock.js:10]: because I'm adding a classname for the RHEL9 content to add the margin

Comment (adonispuente) [reply src/Components/RegAssistCodeBlock/RegAssistCodeBlock.js:10]: woops, I only saw the classname on the first instance not the second. a nitpick would be just to change the name to something like pfclass or attribute but not neccessary

**Review by adonispuente — APPROVED** [review]

**Pull request merged**